src/browser/rollbar.js

```diff
@@ -87,6 +91,9 @@ Rollbar.log = function() {
 Rollbar.prototype.debug = function() {
   var item = this._createItem(arguments);
   var uuid = item.uuid;
+  if (this._sameAsLastError(item)) {
```

Comment:
If we transition to having all of the `debug/info/warn/...` methods always call `this.client.log(item, level)` we can abstract this code into the `client.log()` method.